packages/block-editor/src/components/block-toolbar/style.scss

```
@@ -37,15 +37,20 @@
 		border-right: $border-width solid $gray-300;
 	}
 
+	&.is-synced,
 	&.is-connected {
-		.block-editor-block-switcher .components-button::before {
+		.block-editor-block-switcher .components-button::after {
+			content: "";
+			display: block;
+			position: absolute;
+			left: 2px;
+			height: $grid-unit-40;
+			width: $grid-unit-40;
+			z-index: -1;
 			background: color-mix(in srgb, var(--wp-block-synced-color) 10%, transparent);
```

```suggestion
			background: rgba(var(--wp-block-synced-color--rgb), 0.1);
```

This is not a blocker and is not directly related to this PR, but if we use the rgb variable, we don't need color-mix.

---

Always appreciate your reviews and input. Agree it's separate, though I wanted to comment that I quite like color-mix in this case, because it allows us to have a single accent color, rather than multiple, and still use it in opacity versions. I'd actually consider going the other direction, replacing instances of --wp-block-synced-color--rgb with a color-mix version.